pages/dkp/kommander/1.3/projects/platform-services/index.md

```diff
@@ -167,6 +167,6 @@ status:
 
 The Konvoy Addon Controller is then responsible for deploying the application.
 
-After the addon is deployed, the addon may be upgraded to a newer version. The only versions shown are compatible with the clusters in that project.
+After the platform service is deployed, the platform service may be upgraded to a newer version. The only versions shown are compatible with the clusters in that project.
 
-![Project addon edit form](/dkp/kommander/1.3/img/project-catalog-addon-edit.png)
+![Project platform service edit form](/dkp/kommander/1.3/img/project-catalog-addon-edit.png)
```

Comment:
Do we have to worry about changing the name of the img files and stuff like that, or are we good? I'm leaning toward this is okay to me. More of a @cmastr / docs management question, I suppose.

Reply:
no on image files.